### PR TITLE
wichita new data integration

### PR DESCRIPTION
**Pull request by mgerchick**
integrate 2018-2020 data

### CODE REVIEW

File: lib/states/ks/wichita.R [hunk 1]
```diff
@@ -58,14 +58,17 @@ clean <- function(d, helpers) {
         "pedestrian",
         "vehicular"
       ),
+      citation_date_time = sub("\\+.*", "", citation_date_time),
```
Comment (jgaeb) [inline lib/states/ks/wichita.R:61]: Can you add a comment explaining what's going on here?

**Review by jgaeb — CHANGES_REQUESTED** [review]
Great! Looks like this was hopefully pretty straight forward. Only one small change.

File: lib/states/ks/wichita.R (hunk 1 above)
Comment (mgerchick) [reply lib/states/ks/wichita.R:66]: see comment added

**Review by jgaeb — COMMENTED** [review]
Looks good to me!

**Review by jgaeb — APPROVED** [review]

File: lib/states/ks/wichita.R [hunk 2]
```diff
@@ -58,14 +58,22 @@ clean <- function(d, helpers) {
         "pedestrian",
         "vehicular"
       ),
+      # NOTE: some of the date info in the new 2018-2020 data
+      # has what appears to be some sort of timezone info at the end
+      # of the string - e.g., "2018/01/01 10:48:00+00". All 
+      # entries with this format have "+00" at the end, which we remove 
+      # for processing
+      citation_date_time = sub("\\+.*", "", citation_date_time),
```
Comment (KenielYao) [inline lib/states/ks/wichita.R:66]: I'd add a "$" at the end of the regex, just in case, but no big deal.

**Review by KenielYao — APPROVED** [review]

File: lib/states/ks/wichita.R (hunk 2 above)
Comment (mgerchick) [reply lib/states/ks/wichita.R:66]: good suggestion!

**Pull request merged**